terraspace/app/stacks/event/locals.tf

```diff
@@ -0,0 +1,7 @@
+locals {
+  environment_variables = {
+    "NODE_ENV" = var.node_env
```

Comment:
```suggestion
    "NODE_ENV" = var.node_env
    "EVENT_TARGET" = "https://webhook.site/e3781bb4-42be-4c61-b9d1-173ad18efaed"
```

Reply:
I've added EVENT_TARGET value for staging and prod envs in these files:
terraspace/app/stacks/event/tfvars/us-west-2/staging.tfvars
terraspace/app/stacks/event/tfvars/us-west-2/prod.tfvars